### PR TITLE
Dahl tabs

### PR DESCRIPTION
**Pull request by pswk1**
Adjustments to tabs styling to match Figma specifications

### CODE REVIEW

File: src/components/FeatureViews/Accounts/UserProfile.js [hunk 1]
```diff
@@ -2,13 +2,13 @@ import React, { useCallback, useEffect, useMemo, useState } from "react";
...
 import { useSelector } from "react-redux";
 
-import Badge from "@material-ui/core/Badge";
 import BioIcon from "@material-ui/icons/PersonOutlined";
 import ContactIcon from "@material-ui/icons/ContactPhoneOutlined";
 import CoursesIcon from "@material-ui/icons/SchoolOutlined";
 import CurrentSessionsIcon from "@material-ui/icons/AssignmentOutlined";
 import Grid from "@material-ui/core/Grid";
 import Hidden from "@material-ui/core/Hidden/Hidden";
+import { LabelBadge } from '../../../theme/ThemedComponents/Badge/LabelBadge';
```
Comment (andrewjknapp) [inline src/components/FeatureViews/Accounts/UserProfile.js:11]: I would recommend moving this to the bottom of your import statements to group all the material-ui imports together. Also use double quotes to match the other imports.

**Review by andrewjknapp — CHANGES_REQUESTED** [review]
Hey Peter, I found a couple places in Omou where the old tabs are still there. These include the EnrollmentView.js and the class display after going through the Registration tab. I also was not sure how to test to see the LabelBadge that you added in. Let me know when you have updated these and I can finish this review. Thanks
image
image

**Comment by pswk1** [comment]
@dhuangg @CalvinFronda 
Tagging here because Dahl tab migration (particularly Notes) is waiting on REST to GraphQL migration in Accounts &  EnrollmentView

**Review by andrewjknapp — APPROVED** [review]
You have addressed all of my comments, looks like this is good to merge.

**Pull request merged**